content/articles/data-representation-using-brython/index.md

```diff
@@ -0,0 +1,316 @@
+[87, 121],[76, 136],[102, 189],[71, 149],[71, 144],[140, 177],[120, 82],[75, 70],[149, 149],[134, 179],
+[78, 127],[82, 112],[112, 185],[124, 137],[179, 91],[71, 124],[103, 172],[118, 176],[77, 166],[85, 106],
+[152, 153],[101, 174],[174, 111],[94, 95],[97, 167],[163, 154],[146, 110],[76, 185],[77, 183],[152, 135],
+[168, 161],[168, 156],[106, 165],[100, 81],[155, 167],[106, 161],[146, 118],[93, 184],[102, 79],[144, 106],
+[135, 188],[99, 150],[73, 128],[138, 139],[81, 72],[140, 147],[112, 127],[87, 94],[174, 139],[77, 130],
+[131, 95],[135, 111],[127, 179],[79, 162],[131, 109],[108, 143],[109, 92],[108, 158],[110, 102],[79, 149],
+[92, 141],[86, 190],[71, 99],[129, 108],[123, 116],[89, 86],[132, 181],[137, 190],[132, 143]]
+
+}, {
+'name': 'Hispanic',
+'color': 'green',
+'data': [[92, 115],[136, 109],[80, 74],[135, 95],[126, 132],[163, 110],[73, 144],[154, 74],[150, 154],
+[178, 168],[99, 148],[116, 143],[95, 171],[117, 90],[119, 170],[133, 156],[93, 137],[102, 148],[118, 155],
+[154, 89],[161, 78],[98, 104],[153, 156],[70, 130],[76, 141],[125, 82],[169, 180],[158, 142],[110, 108],[71, 136],
+[168, 155],[145, 96],[167, 160],[85, 172],[115, 84],[165, 81],[123, 184],[132, 175],[110, 168],[173, 133],[94, 127],
+[81, 82],[171, 132],[96, 121],[75, 105],[140, 151],[83, 169],[118, 159],[155, 130],[180, 142],[171, 70],[172, 87],
+[164, 133],[71, 83],[163, 118],[160, 112],[152, 78],[115, 83],[158, 105],[103, 167],[109, 121],[151, 147],[83, 72],
+[169, 103],[130, 110],[171, 141],[86, 76],[82, 186],[163, 80],[162, 147],[108, 188],[152, 158],[146, 176],[139, 107],
+[109, 153],[114, 138],[119, 172],[109, 104],[99, 87],[104, 109],[88, 189],[73, 95],[138, 134],[109, 101],[179, 185],
+[105, 129],[140, 85],[133, 98],[157, 165],[152, 159],[179, 154],[175, 165],[119, 187],[70, 156],[87, 184],[73, 181],
+[165, 131],[146, 114],[177, 176],[100, 106],[140, 101],[168, 109],[95, 72],[137, 124],[160, 182],[121, 142],[154, 128],
+[156, 183],[99, 149],[132, 78],[77, 110],[108, 83],[110, 70],[166, 125],[150, 115],[83, 88],[81, 176],[155, 112],[119, 70],
+[102, 158],[108, 147],[169, 153],[135, 121],[143, 93],[134, 128],[167, 96],[140, 138],[88, 146],[76, 157],[176, 91],[170, 108],
+[79, 158],[148, 75],[145, 138],[82, 186],[102, 118],[96, 146],[162, 119],[76, 138],[129, 145],[116, 103],[89, 137],[169, 73],[118, 144],
+[96, 155],[133, 164],[168, 138],[154, 110],[173, 80],[125, 164],[85, 94],[142, 182],[91, 183],[81, 135],[116, 180],[149, 119],[168, 110],
+[103, 110],[113, 181],[95, 184],[134, 80],[104, 71],[78, 125],[103, 127],[139, 81],[115, 181],[127, 111],[91, 146],[77, 161],[80, 143],
+[135, 86],[156, 155],[171, 167],[101, 184],[121, 75],[79, 157],[90, 180],[117, 167],[147, 78],[145, 157],[114, 141],[141, 134],[122, 147],
+[156, 177],[177, 83],[180, 145],[122, 106],[100, 147],[155, 72],[85, 81],[90, 122],[107, 161],[73, 168],[100, 104],[117, 82],[153, 71],[88, 74],
+[171, 117],[83, 185],[125, 136]]
```

Comment:
To do that, it will involve the use of pandas. At the moment pandas library is not working with brython, thus, not making this possible. i can reduce the data so it won't be that cumbersome